### DISCUSSION TITLE
Logs: Download `raw` format

### DISCUSSION THREAD

**Discussion by svennergr** [grafana/grafana#61155] [Observability]
Currently logs are downloaded containing stats, labels and timestamps. 
However having a `raw` format, just downloading the plain loglines/entries would be good.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82864. Feel free to continue the discussion around this there. Thank you!
